### PR TITLE
Fix color prop missing for icon, add not selected color for tabs

### PR DESCRIPTION
**Pull request by BogdanDarius**
Issue: Missing color property for icon.
icon_problem

Suggestion: Add property `notselectedcolor`. Use case: if the tabs background is white, the tabs that are not selected will also have white text (it is hardcoded) and you cannot see the text. Would be nice if there was a `notselectedcolor` property for the tabs so you can customize the text color when a tab is not selected.

I have added a pull request for these two things! Have a nice day!

### CODE REVIEW

File: src/components/Tabs/TabButton.svelte [hunk 1]
```diff
@@ -9,6 +9,7 @@
   export let to = "";
   export let selected = "";
   export let color = "primary";
+  export let notselectedcolor = "white";
```
Comment (matyunya) [inline src/components/Tabs/TabButton.svelte]: Let's keep prop naming consistent. `notSelectedColor`?

Comment (BogdanDarius) [reply src/components/Tabs/TabButton.svelte]: Yes sure!
Will you make the modifications or should I update de PR?

Comment (matyunya) [reply src/components/Tabs/TabButton.svelte]: Please post an update, I'd probably need to fork your fork otherwise.

Comment (BogdanDarius) [reply src/components/Tabs/TabButton.svelte]: Updated in the new commit.

Comment (matyunya) [reply src/components/Tabs/TabButton.svelte]: Thanks!

Comment (BogdanDarius) [reply src/components/Tabs/TabButton.svelte]: My pleasure, I love this project!
Any idea when you are gonna push these changes to npm?

Comment (matyunya) [reply src/components/Tabs/TabButton.svelte]: Let me do a minor release.

Comment (matyunya) [reply src/components/Tabs/TabButton.svelte]: Thanks!

Comment (matyunya) [reply src/components/Tabs/TabButton.svelte]: Let me try to fix treeviews in the same release first though.

Comment (BogdanDarius) [reply src/components/Tabs/TabButton.svelte]: Awesome!

**Pull request merged**